python/ray/experimental/dag/class_node.py

```diff
@@ -81,7 +81,10 @@ def _contains_input_node(self) -> bool:
     def __getattr__(self, method_name: str):
         # User trying to call .bind() without a bind class method
         if method_name == "bind" and "bind" not in dir(self._body):
-            raise AttributeError(f".bind() cannot be used again on {type(self)} ")
+            raise AttributeError(
+                f".bind() cannot be called again on {type(self)}. "
+                "You should specify a method (`node.method_name.bind()`) instead."
```

Comment:
specify a method -> specify a class method ?

Reply:
class method has another meaning: `@classmethod` decorated method. 

Method is the right one I think to mean "instance method"